### ISSUE TITLE
Blank space above each picture

### ISSUE THREAD

**Issue by lanhuaguizha** [noties/Markwon#188]
* **Markwon Version**: _{4.2.0}_

1. Picture loads from remote firebase server
2. My raw content is below

> Tell me why this is happening
![image](https://firebasestorage.googleapis.com/v0/b/christian-1234.appspot.com/o/storage%2Femulated%2F0%2FDownload%2Fbeautiful-asian-girls-wallpapers21.jpg?alt=media&token=5354a845-5b54-4fdd-8ad9-16f83989e147)
I don't know how to make it work 😭
![image](https://firebasestorage.googleapis.com/v0/b/christian-1234.appspot.com/o/storage%2Femulated%2F0%2FDownload%2Fbeautiful-asian-girls-wallpapers21.jpg?alt=media&token=2a3413d7-980a-4c52-8c9c-419597d06827)
![image](https://firebasestorage.googleapis.com/v0/b/christian-1234.appspot.com/o/storage%2Femulated%2F0%2FDownload%2Fbeautiful-asian-girls-wallpapers21.jpg?alt=media&token=d4420e19-0aaf-465a-9ce5-ae380e88a9d7)
![image](https://firebasestorage.googleapis.com/v0/b/christian-1234.appspot.com/o/storage%2Femulated%2F0%2FDownload%2Fbeautiful-asian-girls-wallpapers21.jpg?alt=media&token=73228f62-afec-4a75-8968-2b39b57ce9ec)
**This is the image**
![image](https://firebasestorage.googleapis.com/v0/b/christian-1234.appspot.com/o/storage%2Femulated%2F0%2FDownload%2Fbeautiful-asian-girls-wallpapers21.jpg?alt=media&token=d7a21c48-8df9-4494-b654-9b3996fad274)
![image](https://firebasestorage.googleapis.com/v0/b/christian-1234.appspot.com/o/storage%2Femulated%2F0%2FDownload%2Fbeautiful-asian-girls-wallpapers21.jpg?alt=media&token=423268f5-71e8-4be1-b04e-3c5aedbe12a5)
![image](https://firebasestorage.googleapis.com/v0/b/christian-1234.appspot.com/o/storage%2Femulated%2F0%2FDownload%2Fbeautiful-asian-girls-wallpapers21.jpg?alt=media&token=673a76a0-0d3a-4141-898f-99dee6c00dd9)
3. But Markwon showed it like this(issue:Yellow blank space above each picture )
![1](https://user-images.githubusercontent.com/9106780/72327428-0ad2f280-36ec-11ea-81f2-6ee9e3243d6b.png)
4. And I code like this, but did not fix the problem
![ECD1C198-FD38-4eff-B7E9-BB689FD7EC9F](https://user-images.githubusercontent.com/9106780/72327669-8339b380-36ec-11ea-8abd-91e6be24f7fc.png)
5. Can you help me or provide some ideas. I've reviewed all the issues and didn't find mine. Thank you

**Comment by noties** [comment]
Hello @lanhuaguizha !

Unfortunately I could not reproduce this issue on my setup. Can you please share you layout XML? It seems that something is messing with TextView's internal coordinate system.

BTW, you don't need both `ImagesPlugin` and `GlideImagesPlugin`, only the last one would be used.

**Comment by lanhuaguizha** [comment]
@noties Thank you very much for your reply. 
1. My XML layou is 
```
<TextView
                android:id="@+id/textView"
                style="@android:style/TextAppearance.Large"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:lineSpacingMultiplier="1.2"
                android:textIsSelectable="true"
                android:layout_margin="@dimen/activity_horizontal_margin"
```
2. This time I deleted my ImagePlugin line code
3.  Yes, configureSpansFactory code was deleted.

But the problem is still. This is my submission https://github.com/lanhuaguizha/Christian/commit/a34948b3ae9de2b1ed743980f275474be9e8e66a.
I think you can reivew my code, or pull the project. See if it's a layout problem. I can't see the problem by myself

**Comment by lanhuaguizha** [comment]
@noties Sorry, I think I see where the problem is, the layout
`android:lineSpacingMultiplier="1.2"`
It caused the problem, and I deleted that fix the issue.

**Comment by noties** [comment]
@lanhuaguizha does it mean this issue is resolved and we can close it? 😉

**Comment by lanhuaguizha** [comment]
Yes. Thank you

**Comment by benvium** [comment]
We ran into this issue today as well. Like @lanhuaguizha saw,  `android:lineSpacingMultiplier` being in a style used by the TextView containing Markwon caused the issue. 

In our case, removing `lineSpacingMultiplier` made the text harder to read, so we used `android:lineSpacingExtra` instead and the unexpected padding was much reduced. 
As a guess (I haven't checked the code) I'd assume that the image height is being multiplied by `lineSpacingMultiplier` when calculating the space needed by the image, thus causing the blank spaces we saw.
